### PR TITLE
djs: revised template to allow for subdomains in the domain name

### PR DESCRIPTION
**Pull request by dylanjshaw001**

### CODE REVIEW

File: template.tpl [hunk 1]
```diff
@@ -33,18 +33,15 @@ ___TEMPLATE_PARAMETERS___
...
     "name": "site_name",
     "simpleValueType": true,
-    "help": "If your site is xyz.com please enter xyz in this field.",
     "valueValidators": [
       {
-        "type": "REGEX",
-        "args": [
-          "^[^.|{]*$"
-        ],
-        "errorMessage": "It looks like you entered a URL.\n\nPlease only enter the domain name. If your site is https://xyz.com, please only enter xyz here.",
```
Comment (raymondborkowski) [inline template.tpl:43]: Should there be an error message like this for our new regex?

File: template.tpl [hunk 2]
```diff
@@ -33,18 +33,15 @@ ___TEMPLATE_PARAMETERS___
     "type": "TEXT",
     "name": "site_name",
     "simpleValueType": true,
-    "help": "If your site is xyz.com please enter xyz in this field.",
```
Comment (raymondborkowski) [inline template.tpl:36]: Should we have a help message for this field like we did previously?

File: template.tpl [hunk 3]
```diff
@@ -70,14 +67,34 @@ const createQueue = require('createQueue');
...
     trigger: 'set',
-    data: {
+    data: {  
         member_id: data.member_id || ''
     }
 });
 
-let site_name = data.site_name;
 
-injectScript('https://init.blackcrow.ai/js/core/' + encodeUriComponent(site_name) + '.js?source=gtm&version=template');
```
Comment (raymondborkowski) [inline template.tpl:80]: I think we want to keep source=gtm and version=template

Comment (dylanjshaw001) [reply template.tpl:80]: @raymondborkowski  Sandbox doesn't like query string here anymore, not sure why...

image

File: template.tpl (hunk 2 above)
Comment (dylanjshaw001) [reply template.tpl:36]: @raymondborkowski, not sure how this one got removed or how to add it in the UI but i will add it to the template after export.

image

File: template.tpl (hunk 1 above)
Comment (dylanjshaw001) [reply template.tpl:43]: @raymondborkowski , added these back in (see screenshot on other comment)

Comment (raymondborkowski) [reply template.tpl:43]: I don't think I am seeing the changes reflected in this PR FYI @dylanjshaw001

**Review by raymondborkowski — APPROVED** [review]

**Pull request merged**